### ISSUE TITLE
Reference error in types.js

### ISSUE THREAD

**Issue by codecandies** [dicebear/dicebear#123]
Hi,

I'm trying to use the micah avatars with

```html
<script>
    import { createAvatar } from '@dicebear/avatars';
    import * as style from '@dicebear/micah';

    let svg = createAvatar(style, {
        seed: 'abcdefgh',
        backgroundColor: '#ffffff',
        dataUri: true
    });
</script>
```

The packages are installed. The node version is 15.8.0.

in a Svelte App, but I directly get an error:
`ReferenceError: Can't find variable: exports` in `node_modules/@dicebear/avatars/lib/types.js`

(It worked last week with the old API and the robotts sprites. And thanks for this great work nevertheless…)

**Comment by FlorianKoerner** [comment]
I can reproduce the problem in svelte. I'll take a closer look.

**Comment by FlorianKoerner** [comment]
Just published a release candidate for version 4.7 with a fix. Thanks for your bug report.

```
npm install @dicebear/avatars@next @dicebear/avatars-micah@next
```

**Comment by codecandies** [comment]
Tried to install, but:
https://github.com/dicebear/dicebear/compare/v4.7.0-alpha.1...main#diff-c38d3105d4f6dc43ddb30a0222b9c000335deb11aea2b6d7a6ac28c9d2777a7fR62 

`"@dicebear/avatars": "^4.6.1"` leads to an npm error…

**Comment by FlorianKoerner** [comment]
Replace the versions of @dicebear/avatars and @dicebear/micah in the package.json with "4.7.0-rc.0" for now and do an `npm install` afterwards. That should solve your problem.

```
"@dicebear/avatars": "4.7.0-rc.0",
"@dicebear/micah": "4.7.0-rc.0",
```

**Comment by FlorianKoerner** [comment]
Ok, I see what you mean. You can ignore this npm error for now. It should still work. Fix will come tonight.

**Comment by codecandies** [comment]
> Works with yarn btw.
:D

**Comment by FlorianKoerner** [comment]
This should work:

```
  "dependencies": {
    "@dicebear/avatars": "^4.7.0-rc.0",
    "@dicebear/micah": "^4.7.0-rc.1"
  }
```

**Comment by FlorianKoerner** [comment]
Fixed in version 4.7.0
